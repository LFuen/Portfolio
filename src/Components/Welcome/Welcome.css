@keyframes animated-border {
    0% {
        box-shadow: 0 0 0 0 rgba(255, 255, 255, 0.589);
    }
    100% {
        box-shadow: 0 0 0 20px rgba(255, 255, 255, 0);
    }
}

#versed:hover {
    box-shadow: 0 0 6px 6px rgba(114, 215, 255, 0.5);
    animation: animated-border 2s infinite;
}

.margin {
    margin-top: 100px !important;
}