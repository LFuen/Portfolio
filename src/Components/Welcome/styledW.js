import {fadeInDown} from 'react-animations'
import styled, {keyframes} from 'styled-components'

const fadeIn = keyframes`${fadeInDown}`

const WelcomeDiv = styled.div`
    animation: 2s ${fadeIn};
`
export {
    WelcomeDiv
}