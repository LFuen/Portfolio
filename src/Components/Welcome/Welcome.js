import React, {Component} from 'react'
import './Welcome.css'


class Welcome extends Component{
    render(){
        return (
        <div>
            <header id='Top' className='margin'>
                <h1 className='flexRow' >Lili Fuentes</h1>
            </header>      
            <div className='margin'> 
                <span title='Say Cheese!'><img className='centerImage' alt='Hiking' src='https://lfuen.github.io/portimages/smile.jpg'/></span> 
            </div>
            <div>
                <h2 className='flexRow' id='Welcome'>Welcome</h2>
                    <div className='flexColumn'>
                    <p>My name is Lili and I want to thank you for visiting my page. Let me take a minute here to share what it is that I do.</p>
                    <p>My official title is Software Engineer, but I'm versed in:</p>
                    </div>
                    <div className='borderOne' id='versed'>
                    <p><b>Front-end Developing</b></p>
                    <p><b>Back-end Developing</b></p> 
                    <p><b>Full-Stack Developing</b></p> 
                    <p><b>React / JavaScript / HTML / CSS / jQuery / Node</b></p>
                    </div>
            </div>
        </div>
        );
    }
}

export default Welcome