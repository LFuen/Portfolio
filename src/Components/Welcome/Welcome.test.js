import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Welcome from './Welcome'


describe(`Welcome component`, () => {
    describe(`Snapshot`, () => {
        it(`renders the UI as expected`, () => {
            const welcome = renderer.create(
                <BrowserRouter>
                    <Welcome />
                </BrowserRouter>).toJSON()
            expect(welcome).toMatchSnapshot()
        })
    })

    describe(`Smoke test`, () => {
        it(`renders without crashing`, () => {
            const div = document.createElement('div')

            ReactDOM.render(
                <BrowserRouter>
                    <Welcome />
                </BrowserRouter>,
                div
            )
            ReactDOM.unmountComponentAtNode(div)
        })
    })
})