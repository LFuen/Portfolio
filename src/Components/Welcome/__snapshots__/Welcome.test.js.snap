// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component Snapshot renders the UI as expected 1`] = `
<div>
  <header
    className="margin"
    id="Top"
  >
    <h1
      className="flexRow"
    >
      Lili Fuentes
    </h1>
  </header>
  <div
    className="margin"
  >
    <span
      title="Say Cheese!"
    >
      <img
        alt="Hiking"
        className="centerImage"
        src="https://lfuen.github.io/portimages/smile.jpg"
      />
    </span>
  </div>
  <div>
    <h2
      className="flexRow"
      id="Welcome"
    >
      Welcome
    </h2>
    <div
      className="flexColumn"
    >
      <p>
        My name is Lili and I want to thank you for visiting my page. Let me take a minute here to share what it is that I do.
      </p>
      <p>
        My official title is Software Engineer, but I'm versed in:
      </p>
    </div>
    <div
      className="borderOne"
      id="versed"
    >
      <p>
        <b>
          Front-end Developing
        </b>
      </p>
      <p>
        <b>
          Back-end Developing
        </b>
      </p>
      <p>
        <b>
          Full-Stack Developing
        </b>
      </p>
      <p>
        <b>
          React / JavaScript / HTML / CSS / jQuery / Node
        </b>
      </p>
    </div>
  </div>
</div>
`;
